docs/source/results.rst

```diff
@@ -80,10 +80,6 @@ The `TrainResult` basic usage is this:
 
 minimize
 ^^^^^^^^
-When using TrainResult, the metric that needs to be minimized is passed to this
```

Comment:
why this doc is removed?

Reply:
Sorry. I think I oversaw this change, when doing the rebase. I will fix this.